framework/src/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -1195,6 +1195,10 @@ public AnnotatedTypeMirror getAnnotatedTypeLhs(Tree lhsTree) {
             case MEMBER_SELECT:
             case ARRAY_ACCESS:
                 res = getAnnotatedType(lhsTree);
+                if (dependentTypesHelper != null && lhsTree.getKind() == Kind.VARIABLE) {
```

Comment:
Is this really the right place to perform this standardization? Shouldn't it be part of `getAnnotatedType`?